pkg/sql/logictest/testdata/logic_test/crdb_internal

```diff
@@ -760,6 +760,15 @@ SELECT * FROM crdb_internal.create_type_statements
 54  test  public  66  enum1  CREATE TYPE public.enum1 AS ENUM ('hello', 'hi')  {hello,hi}
 54  test  public  68  enum2  CREATE TYPE public.enum2 AS ENUM ()               {}
 
+query ITTITTT
+SELECT * FROM crdb_internal.create_type_statements WHERE descriptor_id = (('enum1'::regtype::oid::int) - 100000)::oid
+----
+54  test  public  66  enum1  CREATE TYPE public.enum1 AS ENUM ('hello', 'hi')  {hello,hi}
+
+query ITTITTT
+SELECT * FROM crdb_internal.create_type_statements WHERE descriptor_id = 'foo'::regclass::oid
+----
+
 # Test the virtual index as well.
```

Comment:
my fav part of this is the pre-existing comment here :P

Reply:
ooh moved this yp!